cmake/Modules/VersionConfig.cmake

```diff
@@ -32,6 +32,13 @@ if(NOT DEFINED OBS_VERSION_OVERRIDE)
         string(REPLACE "." ";" _CANONICAL_SPLIT ${_CANONICAL_SPLIT})
         list(GET _CANONICAL_SPLIT 0 1 2 _OBS_VERSION_CANONICAL)
         string(REPLACE "." ";" _OBS_VERSION ${_OBS_VERSION})
+        # Get 8-character commit hash without "g" prefix
+        foreach(VERSION_PART ${_CANONICAL_SPLIT})
+          if(VERSION_PART MATCHES "^g")
+            string(SUBSTRING ${VERSION_PART}, 1, 8, OBS_COMMIT)
+            break()
+          endif()
+        endforeach()
```

Comment:
We use 9-character hashes in CI. Do we want to use 9-characters here?

Reply:
I cannot remember if this had a specific reason behind it. In theory it shouldn't matter.

Reply:
Can probably be adjusted later anyway if need be.